test/libp2p/protocol/kademlia/peer_routing_table_test.cpp

```diff
@@ -54,9 +54,8 @@ TEST_F(PeerRoutingTableTest, BusWorks) {
 
   std::unordered_set<PeerId> peerset;
 
-  auto addHandle = addCh.subscribe([&](const PeerId &pid) {
-    peerset.insert(pid);
-  });
+  auto addHandle =
+      addCh.subscribe([&](const PeerId &pid) { peerset.insert(pid); });
```

Comment:
Please, use clang-format tool to be in according with using code-style. We are using Google code style, defined in file [.clang-format](https://github.com/libp2p/cpp-libp2p/blob/master/.clang-format)